plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/BaseElasticsearchConnectorTest.java

```diff
@@ -1468,6 +1468,21 @@ public void testFilters()
                 .put("ipv6_column", "2001:db8:0:0:1:0:0:1")
                 .buildOrThrow());
 
+        assertQuery("SELECT count(*) FROM filter_pushdown WHERE keyword_column <> ''", "VALUES 1");
```

Comment:
Add a single-line comment to indicate the test cases about. like `// _score column`

Reply:
Thanks for suggestion, added the comment